addons/bestja_estimation_reports/models.py

```diff
@@ -89,6 +89,24 @@ class EstimationReportEntry(models.Model):
         store=True,  # Needed by graph view
         related='estimation_report.responsible_organization',
     )
+    responsible_project = fields.Many2one(
+        'bestja.project',
+        string=u"Projekt super nadrzędny",
```

Comment:
First word in this label is capitalized. This is inconsisten with the adjacent added fields.